proto/backup.proto

```diff
@@ -43,6 +43,11 @@ message BackupMeta {
 
     // In incremental backup, DDLs which are completed in (lastBackupTS, backupTS] will be stored here.
     bytes ddls = 10;
+
+    // Save the br commit to run backup jobs
+    string br_version = 11;
+    // A map of the tikv store ID to store version
+    map<uint64, string> tikv_versions = 12;
```

Comment:
I don't think we need this, it should be the same as cluster version.

Reply:
What if different TiKVs have different versions? By the way, what should I fill in `cluster version`?

Reply:
> What if different TiKVs have different versions?

It's abnormal, or a cluster is in rolling upgrade.

> By the way, what should I fill in cluster version?

https://docs.pingcap.com/tidb/stable/pd-control#:~:text=cluster-version (the version is reported by TiKV)